languages/julia/exercises/concept/methods.basic/.docs/introduction.md

```diff
@@ -0,0 +1,40 @@
+The entire Julia track will require you to treat your solution like a mini-libraries, i.e. you need to define functions, types etc. which will then be run against a test suite.
+For that reason, we will introduce basic, named functions as the very first concept.
+
+### Defining functions
+
+There are two common ways to define a named function in Julia:
+
+```julia
+function muladd(x, y, z)
+    return x * y + z
+end
+```
+
+and
+
+```julia
+muladd(x, y, z) = x * y + z
+```
+
+The latter is most commonly used for one-line function definitions or mathematical functions.
+
+### Invoking functions
+
+Invoking a method is done by specifying its name and passing arguments for each of the method's parameters:
```

Comment:
The text is referring to a method, but previously you mention a function is being defined. Are both names allowed?

Reply:
Thanks for pointing this out. They're different things. I think technically "method" makes more sense here but I'll switch it to function to keep things simpler for now (the manual first defines functions and then later on goes into details about methods as well)